### ISSUE TITLE
Add an option in the "trail always on" hack to disable it automatically when the icon is hidden

### ISSUE THREAD

**Issue by Sloweneuh** [absoIute/Mega-Hack-Pro-Future#170]
It feels out of place to have a trail in a cinematic like at the end of Espionage. Is it possible to add an option to detect the hide trigger and disable the trail when it's active ?

**Comment by Brittank88** [comment]
I agree with this. Also would love to be able to toggle between it working during normal play, in the editor, or both.

**Comment by absoIute** [comment]
Will add this as default behaviour soon
